firmware/Board.h

```diff
@@ -6,12 +6,16 @@
 namespace board 
 {
   
+  struct Peripheral_t
+  {
+    int ENABLE;
+    int CURRENT;
+  };
+
```

Comment:
I removed all the power control from the motor objects that we used to use when we powered the motors through the eboard and moved that into the peripheral objects so enable turns on the power and current reports the current draw of the peripheral. if we ever add some kind of current sensing for the motors we can current sensing back into the motor abstraction.